roles/splunk/tasks/configure_license.yml

```diff
@@ -1,4 +1,18 @@
 ---
+- block:
+  - name: Copy licenses to the licensemaster
+    copy:
+      src: "{{ item }}"
+      dest: "{{ splunk_home}}/etc/license/enterprise/{{ item }}"
+      owner: "{{ splunk_nix_user }}"
+      group: "{{ splunk_nix_group }}"
+      mode: 0644
+    loop: "{{ splunk_licenses }}"
+    notify: restart splunk
+  when:
+    - "'licensemaster' in group_names"
+    - splunk_licenses != 'undefined'
```

Comment:
@schneewe Change this to `splunk_licenses|length > 0` or `splunk_licenses.0 is defined`